galaxy.yml

```diff
@@ -50,10 +45,10 @@
       restart_galaxy: true
       when: install_galaxy
       
-    #check if tool-data persisted on galaxy (it does nothing if galaxy_data == galaxy_server_dir)
     - role: galaxy.movedata
-      check_tool_data: yes
-      when: install_galaxy
+      move_to_persistent: true
```

Comment:
Don't force this to be true, you'll only want to do this when supplying an empty export dir.
this will probably also help fix the failing tests.